.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: java
 sudo: false
 jdk:
-  - oraclejdk8
+  - openjdk8
```

Comment:
I'm afraid the test cases that failed at https://travis-ci.org/mjiderhamn/classloader-leak-prevention/jobs/619083836 could be related to the switch. Since some of what that library aims to protect against is in the JDK, I suggest to stick with the Oracle one for JDK 8, unless there are explicit reasons to make the switch?

Reply:
@mjiderhamn I'll make an adjustment to the travis build.  Oracle no longer works on the upgraded travis setup.  Xenial uses only the tools to download and there are license restrictions thus most things are going to openjdk.  I belive this can be set to use the legacy 'trusty' travis build and stay on oracle as it came with those specific distributions.

Reply:
I should add that I initially did not take this off oracle but that failed due to inability to pull the jdks.  The subsequent update to openjdk failed with issue you mentioned.

Reply:
I put this onto trusty (note it is deprecated so not sure how long it will exist).  That resolves jdk 8.  However there is no jdk 11 there for oracle and even though this code base is on openjdk11 prior to this PR, it also fails on travis.